py_swf/clients/decision.py

```diff
@@ -172,6 +173,10 @@ def finish_decision_with_activity(self, task_token, activity_id, activity_name,
         :type identity: string
         :param activity_input: Freeform text of the input for the activity
         :type identity: string
+        :param override_config_dict: Override default config for scheduled activity timeouts.
+        It includes scheduleToCloseTimeout, scheduleToStartTimeout and startToCloseTimeout.
```

Comment:
Should each of these timeouts be arguments, instead of being a dictionary?

Let's clarify that the inputs are not camelCased and let's be clear on the exact timeouts we can override. "It includes" seems to imply there are more.